src/cloud_service.h

```diff
@@ -19,6 +19,9 @@
 #include "Particle.h"
 #include "BackgroundPublish.h"
 
+// default name for incoming Particle.function
+#define CLOUD_DEFAULT_FUNCTION_NAME "cmd"
```

Comment:
What's the difference between this, `CLOUD_DEFAULT_FUNCTION_NAME` and `CLOUD_KEY_CMD`?

Reply:
I can add another comment to clarify that.

`CLOUD_KEY_CMD` is what is used with the JSON formatted commands to say something like `"cmd":"my_cmd_name"`

`CLOUD_DEFAULT_FUNCTION_NAME` is what is used for the `Particle.function` to accept those JSON commands.

Reply:
Then yes, comment or renaming would help to clarify